vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -563,6 +567,41 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
             this.config.autoExceptionInstrumented = true;
         }
 
+        /**
+         * Create a custom "locationchange" event which is triggered each time the history object is changed
+         */
+        if (this.config.enableAutoRouteTracking === true && typeof history === "object" && typeof window === "object") {
+            // Find the properties plugin
+            extensions.forEach(extension => {
+                if (extension.identifier === PropertiesPluginIdentifier) {
+                    this._properties = extension;
+                }
+            });
+
+            history.pushState = ( f => function pushState() {
```

Comment:
We need to test this in IE 9, pushState was added in IE10 would be good to understand how this code behaves in IE9

Reply:
Agreed. I will add nullchecks on the history functions. afaik, SPAs made to work with IE9 are including polyfills, so I should probably just make sure we are not introducing any error logging for those.